package.json

```diff
@@ -1,15 +1,14 @@
 {
-  "name": "react-vis",
-  "version": "1.12.3",
+  "name": "react-vis-master",
```

Comment:
Note this package `react-vis` is now in a subdirectory. This is now the top level package.json for the workspace.